clients/cobol-lsp-vscode-extension/CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 All notable changes to the COBOL Language Support extension are documented in this file.
 Changelog:
 
+## [0.19.1](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.19.0...0.19.1) (2021-06-24)
+
+#### Changed
+* Error message for invalid variable definition
+* Analysis exceptions propagated to Extended API responses
+
+#### Fixed
+* IDMS statements in conditional blocks
+* Access to undefined element if there is no workspace open
+* Error on multiline optional paragraphs
+* Hover for group items
+* Loading copybooks with any encoding
```

Comment:
What does this one actually mean? All encoding types are now supported for Copybooks? Isn't that more a new feature than a fix?

Reply:
There was a bug that copybooks in one-byte national encodings throw an exception while loading, so this is rather a fix.

Reply:
In that case I'd change it to "Error when loading copybooks with one-byte national encoding" - write what's been fixed.

Reply:
That makes sense